tools/project_manager/page_detail.php

```diff
@@ -80,14 +61,14 @@
     {
         if (is_null($round_for_page_selection) )
         {
-            echo sprintf( _("Showing only the pages of user '%s'."), 
-                          $username_for_page_selection );
+            echo sprintf( _("Showing only the pages of user '%s'."),
+                          html_safe($username_for_page_selection) );
         }
         else
         {
-            echo sprintf( _("Showing only the pages of user '%1\$s' in round %2\$s."), 
-                          $username_for_page_selection, 
-                          $round_for_page_selection );
+            echo sprintf( _("Showing only the pages of user '%1\$s' in round %2\$s."),
+                          html_safe($username_for_page_selection),
+                          html_safe($round_for_page_selection) );
```

Comment:
Again, `$round_for_page_selection` is validated, and none of our round IDs include HTML escapes.